Source/Documentation/Manual/options.rst

```diff
@@ -156,8 +156,8 @@ The option *Player's Location* sets the units according to the Windows
 The option *Route* sets the units based on the data in the route files.
 The other options are self-explanatory.
 
-These windows use the abbreviations *stn* for short tons (2000 lb),
-*t* for long tons (2,240 lb) or *tn* for metric tons (tonnes).
+These windows use the abbreviations *t-us* for short tons (2,000 lb),
+*t-uk* for long tons (2,240 lb) and *t* for metric tons (1,000 kg).
```

Comment:
Chris, I'm really sorry for being so picky, but I would have one more comment, please forgive me.

In almost all other non-English languages the comma (,) is used as a decimal separator. For a non native English reader it is very confusing to read numbers with commas in that, because they never know if that is a decimal separator or a thousand separator. Also in mathematics and physics the thousand separators are never used exactly because of the international nature of the science. Could we possibly also follow a rule not using these in the documentation? I would prefer writing 2,000 as 2000, 2,240 as 2240 and 1,000 as 1000, if possible.

Reply:
OK Peter, I'll do my bit for internationalisation.

I've also refined the Hot Start text as posted on the forum.

Thanks,
Chris